app/models/user.ts

```diff
@@ -102,7 +104,13 @@ export default class UserModel extends Model {
   }
 
   get canAccessMembershipBenefits() {
-    return this.isVip || this.hasActiveSubscription || this.teamHasActiveSubscription || this.teamHasActivePilot;
+    return (
+      this.isVip ||
+      this.hasActiveInstitutionMembershipGrants ||
```

Comment:
We've tied membership grants to subscriptions for a reason - so that we don't need to add special cases like this. Every membership grant has an associated subscription. 

Shouldn't have to edit this at all